### PR TITLE
Mindustry#802 isChromebook check

### PR DESCRIPTION
**Pull request by Aidan9381**
A check to see if the Operating System is Chromebook for Chromebook-specific features

from Mindustry issue #802 https://github.com/Anuken/Mindustry/issues/802

### CODE REVIEW

**Comment by Anuken** [comment]
Does this really work? What does `os.name` return when playing on a Chromebook?

**Comment by Aidan9381** [comment]
You'll have to give me a while to properly test, i can get it to properly compile or get what I'm after. Sorry.

File: arc-core/src/io/anuke/arc/util/OS.java [hunk 1]
```diff
@@ -11,11 +11,12 @@
     static public boolean isAndroid = false;
     static public boolean isARM = System.getProperty("os.arch").startsWith("arm") || System.getProperty("os.arch").startsWith("aarch64");
     static public boolean is64Bit = System.getProperty("os.arch").contains("64") || System.getProperty("os.arch").startsWith("armv8");
+    static public boolean isChromebook = System.getProperty("os.name").contains("Chrome");
```
Comment (joshuaptfan) [inline arc-core/src/io/anuke/arc/util/OS.java]: Variable should probably be named "isChromeOS".

**Comment by Anuken** [comment]
Why I'm concerned this won't work:

*Android documentation*
image

*Wikipedia*
image

If these sources are correct, `os.name` will be Linux and thus `isChromeOS` will always be false.

**Pull request closed without merging**